### ISSUE TITLE
error: ‘zip_file_is_seekable’ was not declared in this scope

### ISSUE THREAD

**Issue by marcus682** [google/mount-zip#15]
```
$ make
make -C lib
make[1] : on entre dans le répertoire « /home/marc/mount-zip/lib »
g++ -c  -I/usr/include/fuse -D_FILE_OFFSET_BITS=64  -Wall -Wextra -Wno-sign-compare -Wno-missing-field-initializers -pedantic -std=c++20 -O2 -DNDEBUG data_node.cc -o data_node.o
data_node.cc: In member function ‘bool DataNode::CacheAll(zip_t*, const FileNode&, std::function<void(long int)>)’:
data_node.cc:272:25: error: ‘zip_file_is_seekable’ was not declared in this scope
  272 |   const bool seekable = zip_file_is_seekable(file.get()) > 0;
      |                         ^~~~~~~~~~~~~~~~~~~~
data_node.cc: In member function ‘Reader::Ptr DataNode::GetReader(zip_t*, const FileNode&) const’:
data_node.cc:296:25: error: ‘zip_file_is_seekable’ was not declared in this scope
  296 |   const bool seekable = zip_file_is_seekable(file.get()) > 0;
      |                         ^~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:41 : data_node.o] Erreur 1
make[1] : on quitte le répertoire « /home/marc/mount-zip/lib »
make: *** [Makefile:55 : lib/libmountzip.a] Erreur 2
```

i got this error when try to compiling, what can i do ?

**Comment by fdegros** [comment]
Which version of `libzip` are you building `mount-zip` with?

The function `zip_file_is_seekable` was first introduced in [`libzip` 1.9.0](https://libzip.org/news/release-1.9.0.html), which was released in June 2022. This function was then "fixed" in [`libzip` 1.9.1](https://libzip.org/news/release-1.9.1.html) released a couple of weeks later, in June 2022 as well.

So you need to use `libzip` 1.9.1 or later in order to build and use `mount-zip`. If your system features an older `libzip`, you might need to build `libzip` yourself from the [most recent sources](https://github.com/nih-at/libzip/).

**Comment by fdegros** [comment]
`mount-zip` 1.0.14 can now be built with older versions of `libzip` that don't feature `zip_file_is_seekable`.
